content/departments/engineering/dev/process/incidents/playbooks/ci.md

```diff
@@ -206,6 +206,26 @@ In order to handle problems with the CI, the following elements are necessary:
 
 You can also refer to the [Loom walkthrough "how to find out if a CI failure is a recurring flake"](https://www.loom.com/share/58cedf44d44c45a292f650ddd3547337).
 
+### Builds are not being created on Buildkite
+
+- Severity: _major_
+- Impact: It's possible to merge a PR without going through CI. No builds are produced and it's impossible to deploy the new commits. 
+- Possible causes:
+  - GitHub is experiencing some outage that is affecting webhooks. 
+  - Buildkite is experiencing some outage.
+  - Webhooks that trigger the builds have been deleted.
+
+#### Actions 
+
+1. Inspect [webhooks status](https://github.com/sourcegraph/sourcegraph/settings/hooks) on the `sourcegraph/sourcegraph` repository settings
+  1. If you're not authorized to see this page, ping `@dev-experience-support` or escalate to `@github-owners`.
+  1. Check the status of the webhook, if it's not green, something is wrong.
```

Comment:
```suggestion
  1. Check the status of the webhook, if it's not green, something is wrong. However, if it is green it is no guarantee that the webhook is operating as usual! If GitHub Webhooks is experiencing degraded performance, it might not be emitting events to the endpoint at all any more, and the green status was the last submission before the outage started. See the next step to verify the status of Webhooks.
```